test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file should be added to the gitignore file, when you do this it won't be pushed to your repository.